### ISSUE TITLE
lib folder missing on npm for v2.4.0

### ISSUE THREAD

**Issue by nico-martin** [ratson/react-intl-redux#66]
Hi there

The current Version on npm has no lib folder: https://www.npmjs.com/package/react-intl-redux/v/2.4.0?activeTab=code
But the main parameter in the package.json still points to the lib-Folder.

**Comment by ghuserchris** [comment]
yeah v2.4.0 is broken, can't resolve react-intl-redux in component.

**Comment by tswaters** [comment]
It looks like @ratson didn't run the build command before publishing?

**Comment by baranoden** [comment]
npm install react-intl-redux@2.3.0 works perfectly fine but unfortunately 2.4 is lack of libs folder
